encoding/ssz/htrutils.go

```diff
@@ -141,8 +141,10 @@ func WithdrawalSliceRoot(withdrawals []*enginev1.Withdrawal, limit uint64) ([32]
 	return MixInLength(bytesRoot, bytesRootBufRoot), nil
 }
 
-func transactionRoot(tx []byte) ([32]byte, error) {
-	chunkedRoots, err := PackByChunk([][]byte{tx})
+// ByteSliceRoot is a helper func to merkleize an arbitrary List[Byte, N]
+// this func runs Chunkify + MerkleizeVector
+func ByteSliceRoot(slice []byte) ([32]byte, error) {
```

Comment:
Are there no tests for this method? Surprised you could rename it without breaking tests :scream:

Reply:
hmm your concern here is probably valid I'll double check with kasey and potuz 😅